paper.md

```diff
@@ -0,0 +1,178 @@
applied as biomarkers for healthy and pathological brain processes. Yet, historically the underlying cellular and circuit level generators of MEG/EEG signals have been difficult to infer. This cell and circuit level understanding is critical to develop theories of information processing based on these signals, or to use these techniques to develop new therapeutics for neuropathology. Computational neural modeling is a powerful technique to hypothesize the neural origin of these signals and several modeling frameworks have been developed. Since MEG/EEG recordings are dominated by neocortex, all models are based on simulating neocortical activity, however they widely vary in the level of biophysical details. One class of models known as neural mass models (NMMs) uses simplified representations to simulate net population dynamics, where hypothesized connectivity among neural “nodes” can be inferred from recordings. The Virtual Brain Project [@sanz2013virtual] and Dynamic Causal Modeling from the SPM software [@friston2003dynamic; @litvak2011eeg] are prominent examples of software that implement NMMs. While NMMs are computationally tractable and advantageous for studying brain-wide interactions, they do not provide detailed interpretation of cell and circuit level phenomena underlying MEG/EEG generation. The primary electrical currents that create MEG/EEG sensor signals are known to come from the intracellular current flow and long and spatially aligned cortical pyramidal neuron dendrites [@hamalainen1993magnetoencephalography]. These currents, known as primary current dipoles, produce extracellular electrical and magnetic fields that are picked up outside of the head with MEG/EEG sensors (for a detailed discussion see @neymotin2020human). Further, source localization methods such as MNE-Python estimate the primary electrical currents. As such, models created to study the cell and circuit origin of these signals are designed with detailed pyramidal neuron morphology and physiology, and are often embedded in a full neocortical column model. HNN is one such detailed neocortical column model [@neymotin2020human], and other examples have been employed using the software LFPy [@linden2014lfpy]. A unique feature of HNN is its workflows for interacting with the template neocortical model through layer specific activations to study ERPs and low frequency brain rhythms. HNN also enables direct comparison between simulation output and source localized data in equal units of measure and supports parameter inference. HNN-core was created to maintain all of the functionality of the original HNN software with additional utility (described below) and a clean, well-tested and documented application programming interface (API). Its adoption of open source development standards, including a simplified installation procedure, unit tests, automatic documentation builds, code coverage, and continuous integration enables community development and long term sustainability.
```

Comment:
```suggestion
MEG/EEG are the leading methods to non-invasively study the human brain with millisecond resolution. They have been applied as biomarkers for healthy and pathological brain processes. Yet, historically the underlying cellular and circuit level generators of MEG/EEG signals have been difficult to infer. This cell and circuit level understanding is critical to develop theories of information processing based on these signals, or to use these techniques to develop new therapeutics for neuropathology. Computational neural modeling is a powerful technique to hypothesize the neural origin of these signals and several modeling frameworks have been developed. Since MEG/EEG recordings are dominated by neocortex, all models are based on simulating neocortical activity, however they widely vary in the level of biophysical details. One class of models known as neural mass models (NMMs) uses simplified representations to simulate net population dynamics, where hypothesized connectivity among neural “nodes” can be inferred from recordings. The Virtual Brain Project [@sanz2013virtual] and Dynamic Causal Modeling from the SPM software [@friston2003dynamic; @litvak2011eeg] are prominent examples of software that implement NMMs. While NMMs are computationally tractable and advantageous for studying brain-wide interactions, they do not provide detailed interpretation of cell and circuit level phenomena underlying MEG/EEG generation. The primary electrical currents that create MEG/EEG sensor signals are known to come from the intracellular current flow through long and spatially aligned cortical pyramidal neuron dendrites [@hamalainen1993magnetoencephalography]. These currents, known as primary current dipoles, produce extracellular electrical and magnetic fields that are picked up outside of the head with MEG/EEG sensors (for a detailed discussion see @neymotin2020human). Further, source localization methods such as MNE-Python estimate the primary electrical currents. As such, models created to study the cell and circuit origin of these signals are designed with detailed pyramidal neuron morphology and physiology, and are often embedded in a full neocortical column model. HNN is one such detailed neocortical column model [@neymotin2020human], and other examples have been employed using the software LFPy [@linden2014lfpy]. A unique feature of HNN is its workflows for interacting with the template neocortical model through layer specific activations to study ERPs and low frequency brain rhythms. HNN also enables direct comparison between simulation output and source localized data in equal units of measure and supports parameter inference. HNN-core was created to maintain all of the functionality of the original HNN software with additional utility (described below) and a clean, well-tested and documented application programming interface (API). Its adoption of open source development standards, including a simplified installation procedure, unit tests, automatic documentation builds, code coverage, and continuous integration, enables community development and long term sustainability.
```
Two nitpicks (using the line numbers in the compiled pdf rather than the markdown version):

- L71: "flow __through__ long and spatially aligned..."
- L87: add comma between "integration" and "enables" because this delineates the end of a prepositional phrase.